plugins/GSdx/GSDevice.cpp

```diff
@@ -45,7 +45,7 @@ GSDevice::GSDevice()
 
 GSDevice::~GSDevice()
 {
-	for(auto &t : m_pool) delete t;
+	for(auto t : m_pool) delete t;
```

Comment:
Two reasons:
1) begin `t` of a pointer type (that is necessary given it's usage in `delete t`, so it's without loss of generality), it's useless to "suggest" to access the data in the list remotely by reference. (which the compiler could/would optimize of course). All the other access to the list element infact are by copying the values (returning actual values from iterators instead of keeping the reference to the pointer, which is useless). So it's to keep the code coherent.
2) `FastList` doesn't expose the methods to access by reference the data (which could or should maybe be fixed, if it's non intended).